examples/functions/algolia-document-sync/index.ts

```diff
@@ -6,25 +6,40 @@ import {algoliasearch} from 'algoliasearch'
 const {ALGOLIA_APP_ID = '', ALGOLIA_WRITE_KEY = ''} = env
 
 export const handler = documentEventHandler(async ({event}) => {
-  const {_id, title, hideFromSearch} = event.data
+  const {_id, title, hideFromSearch, operation} = event.data
 
   const algolia = algoliasearch(ALGOLIA_APP_ID, ALGOLIA_WRITE_KEY)
+  if (operation === 'delete') {
+    try {
+      // We are assuming you already have an algolia instance setup with an index called 'posts'
+      // addOrUpdateObject documentation: https://www.algolia.com/doc/libraries/javascript/v5/methods/search/delete-object/?client=javascript
+      await algolia.deleteObject({
+        indexName: 'posts',
```

Comment:
maybe make this a `const ALGOLIA_INDEX_NAME="posts"` constant above the handler so it's easy to customize and avoid targeting two different indices?